opttbx-commons/pom.xml

```diff
@@ -68,6 +68,11 @@
             <groupId>commons-io</groupId>
             <artifactId>commons-io</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-compress</artifactId>
+            <version>1.22</version>
```

Comment:
Is there a reason why you have selected 1.22 and not 1.23.0?

Reply:
No particular reason, I had the impression this was the latest version.